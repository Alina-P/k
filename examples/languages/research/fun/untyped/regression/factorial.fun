letrec factorial x =
  if x<=0
  then 1
  else x * factorial(x - 1)
in factorial(factorial 4)

// 620448401733239439360000
