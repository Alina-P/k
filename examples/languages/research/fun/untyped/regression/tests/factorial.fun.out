<result>
  620448401733239439360000
</result>
